ToDo.Core/Services/INativeViewService.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Xamarin.Forms;
+
+namespace ToDo.Core.Services
+{
+    public interface INativeViewService
+    {
+        int[] GetCoordinates(VisualElement visualElement);
+        int ValidateYPosition(int y, VisualElement visualElement);
```

Comment:
I changed the names and used GetAdjustedYPosition with a return type int (instead of int array).

Commit: 52ede3fd3185ab1dbed1941b8b2e0c956604f146

Reply:
That was not what I meant (renaming the first method), I was only referencing `ValidateYPosition`. But now I see that in this view it looks like I meant both methods... As I said, I'm new to GitHub reviewing, sorry 🙏🏻 
So I don't think `GetAdjustedYPosition` is a good name for the first method, because it gives you the y position, period, not adjusting anything :smile:. I'd rather go with `GetYPosition`